gpkit/fmincon/simpleflight.py

```diff
@@ -0,0 +1,62 @@
+"Minimizes airplane drag for a simple drag and structure model."
```

Comment:
Although it's not super obvious how one would call the original simpleflight.. Do I really have to use `generate_example_tests`?